src/daft-sql/src/modules/hashing.rs

```diff
@@ -83,10 +84,24 @@ impl TryFrom<SQLFunctionArguments> for MinHashFunction {
             })
             .transpose()?
             .unwrap_or(1) as u32;
+
+        let hash_function = args
```

Comment:
can we add tests for this

https://github.com/universalmind303/Daft/blob/e84ed5b5360135c612cab6ed652b6ed2653dd207/tests/sql/test_exprs.py#L43

Reply:
f2d9ad42d6643f970a3e7cc9d0698a28457f888e